locale/ca/site.json

```diff
@@ -0,0 +1,177 @@
+{
+  "title": "Node.js",
+  "author": "Fundació Node.js",
+  "url": "https://nodejs.org/ca/",
+  "locale": "ca",
+  "scrollToTop": "Torna al començament",
+  "reportNodeIssue": "Informe d'un problema de Node.js",
+  "reportWebsiteIssue": "Informe d'un problema al lloc web",
+  "getHelpIssue": "Aconseguir ajuda",
```

Comment:
great @albertvp the PR is already landed in, so I invite you to create new one adding this modifications and more translations (the remaining files)